.env

```diff
@@ -0,0 +1,10 @@
+SECRET_KEY=@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*
+DEBUG=True
+ALLOWED_HOSTS=localhost,127.0.0.1
+DATABASE_ENGINE=django.db.backends.sqlite3
+DATABASE_NAME=db.sqlite3
+LANGUAGE_CODE=en-us
+TIME_ZONE=UTC
+SECRET_KEY=ecret_key_here
+DEBUG=True
+ALLOWED_HOSTS=localhost,127.0.0.1
```

Comment:
Not fixed. The file still remains here in the PR.
<img width="826" alt="image" src="https://github.com/user-attachments/assets/fef020ef-01af-4715-bb7f-f56e3ec74f17" />